content/enterprise/before-installing/os-specific/rhel-requirements.mdx

```diff
@@ -31,7 +31,7 @@ Afterwards, restart the Docker service and verify the newly installed version us
 
 ### Can I use the Docker version in the RHEL Extras repository?
 
-Yes! Just be sure to [modify the default `libseccomp` profile](/enterprise/before-installing#option-3-docker-engine-using-a-modified-libseccomp-profile).
+If you are running Red Hat Enterprise Linux v7, yes. Red Hat Enterprise Linux v8 does not offer docker in the extras repo. If you are running v7 and choose to run docker from extras, just be sure to [modify the default `libseccomp` profile](/enterprise/before-installing#option-3-docker-engine-using-a-modified-libseccomp-profile).
```

Comment:
I'm struggling with the repository name here. Is it actually called RHEL extras? I wonder if we should use the official name which I think is this: Extra Packages for Enterprise Linux (EPEL). But lmk if I'm very wrong :-)  

```suggestion
Yes, if you are running Red Hat Enterprise Linux v7. If you choose to run Docker from `extras`, you must [modify the default `libseccomp` profile](/enterprise/before-installing#option-3-docker-engine-using-a-modified-libseccomp-profile).

Red Hat Enterprise Linux v8 does not offer Docker in the `extras` repository. 
```